alacritty.yml

```diff
@@ -10,7 +10,7 @@
   # each instance of Alacritty. If it is not present, alacritty will
   # check the local terminfo database and use `alacritty` if it is
   # available, otherwise `xterm-256color` is used.
-  #TERM: xterm-256color
+  #TERM: alacritty
```

Comment:
I think this might be a good place to discuss https://github.com/alacritty/alacritty/issues/2487#issuecomment-587455831. Since people are often confused by this variable, and some applications do behave differently with it (I still use `TERM=xterm-256color`, shhhh).

Though I agree, we should be consistent here and have this set to the most specific thing alacritty can default to.

Reply:
> I think this might be a good place to discuss #2487 (comment). Since people are often confused by this variable, and some applications do behave differently with it

I'm not sure what there is to discuss. The only reason why people are confused is because they don't understand how terminal emulators work. If they desire to understand what `TERM` does, then there's plenty of documentation online for them to research.

Reply:
I was thinking that adding a line like "if you don't have the`alacritty` terminfo installed, `xterm-color256` may be used with limited issues."

In my eyes it would also be nice to gove an example where we need our terminfo instead of the generic xterm-color256, but we don't need to go into too much detail.

Why make our users hunt around online to understand our config file?

Reply:
Your clearly missing my point. But I'll just have to look into this myself and maybe come up with a concrete suggestion. Maybe finally fixing my guicolors for vim in the process... last time I went down this rabbit hole I didn't make it too far, maybe I'll fare better this time.